domains/DataCollection/Assets/NewAssets/AssetSupplementary/index.js

```diff
@@ -79,7 +91,7 @@ const AssetSupplementary = ({ selectedAsset, surveyingOrganization }) => {
                     />
                   )}
               </View>
-              {selectedAsset
+              {selectedAsset !== null && selectedAsset !== {}
```

Comment:
@jamccomb92 @jamesgiegerich94 follow this prop all the way up to where it's being defined and study the default data structure. Figure out how to create the logic to check that if that **selectedAsset** data structure has data, show this component.

I'd also check for other references to **selectedAsset** if you plan to change the default state of **selectedAsset**

Reply:
@jamccomb92 can double check your work James!